apps/schools/domains/circle/components/currentCircle/styles/styles.module.scss

```diff
@@ -18,12 +18,16 @@
   }
 
   .colorCountAllQueries {
-    color: #4CD174
+
   }
 
   .colorCountInProgressQueries {
     color: #2698FF
   }
+
+  .colorCountAcceptedQueries {
+    color: #4CD174
```

Comment:
**It’s really important** 

we have to use the convenience of scss. example 
```
:root {
   --primary: #312ffb;
  --white: #fff;
  --black: #000000; 
  } 
```
and use like this

```
.colorCountAcceptedQueries {
   color: var(--black)
   }
```
  
  or 
  
```
$main-color: #f8bbd0;
$accent-color: #6A1B9A;
```
and use like this

```
@include './somePath/somePath/_colors.scss'

.colorCountAcceptedQueries {
   color: $accent-color
   }
```

Reply:
**it's really convenient + necessary, improves readability**

Reply:
All stuff that you noticed is important but refactoring needs to be made is too big for the time. I will create issues on jira and we will resolve them in next releases. OK?

Reply:
https://lamart.atlassian.net/browse/SCHOOL-793

Reply:
?? and ||: https://lamart.atlassian.net/browse/SCHOOL-794